lib/datadog/di.rb

```diff
@@ -10,5 +16,42 @@ module Datadog
   module DI
     # Expose DI to global shared objects
     Extensions.activate!
+
+    class << self
+      attr_reader :code_tracker
+
+      # Activates code tracking. Normally this method should be called
+      # when the application starts. If instrumenting third-party code,
+      # code tracking needs to be enabled before the third-party libraries
+      # are loaded. If you definitely will not be instrumenting
+      # third-party libraries, activating tracking after third-party libraries
+      # have been loaded may improve lookup performance.
+      #
+      # TODO test that activating tracker multiple times preserves
+      # existing mappings in the registry
+      def activate_tracking!
+        @code_tracker ||= CodeTracker.new
+        # & is demanded by steep, code tracker is always not nil here.
+        code_tracker&.start
```

Comment:
Minor: I wonder if steep would be happy with

```suggestion
        (@code_tracker ||= CodeTracker.new).start
```

Reply:
Yes, it seems to do the trick.